GYBITG/NewAccountViewController.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  NewAccountViewController.swift
+//  GYBITG
+//
+//  Created by Student Account on 5/23/19.
+//
+
+import UIKit
+
+class NewAccountViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSource {
+
+    @IBOutlet weak var emailAddress: UITextField!
+    @IBOutlet weak var password: UITextField!
+    @IBOutlet weak var passwordVerify: UITextField!
+    @IBOutlet weak var firstName: UITextField!
+    @IBOutlet weak var lastName: UITextField!
+    @IBOutlet weak var highSchoolName : UITextField!
+    @IBOutlet weak var clubTeam : UITextField!
+
+    @IBOutlet weak var graduationYear: UIPickerView!
+    
+    @IBOutlet weak var state: UIPickerView!
+    
+    var usersRepository = UsersRepository()
```

Comment:
You may want to check out how Juanita and I instantiate our repos in the AppDelegate file. If you do it directly in the view controller, then every time you leave that view controller you will lose the data.